pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -609,11 +609,13 @@ func (c *tenantSideCostController) OnResponse(
 	if isWrite, writeBytes := req.IsWrite(); isWrite {
 		c.mu.consumption.WriteRequests++
 		c.mu.consumption.WriteBytes += uint64(writeBytes)
+		c.mu.consumption.KVRU += float64(c.costCfg.KVWriteCost(writeBytes))
```

Comment:
[nit] Here and below for `KVReadCost`, could compute `c.costCfg.KVWriteCost(writeBytes)` just once and add it to the two separate counters.

Reply:
Done